### PR TITLE
Routers implemented

### PR DESCRIPTION
**Pull request by shazkhan2**
Routers for Teams, members and timeoff. All routers are imported into app.js, which is then run from server.js.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -3,4 +3,11 @@ PGDATABASE='TimeOffDB'
 PGUSER='TimeOffDB_owner'
 PGPASSWORD='wKT0JAWF6BqD'
 ENDPOINT_ID='ep-empty-voice-a2dpidxi'
-PORT=4050
\ No newline at end of file
+PORT=4051
```
Comment (aina21) [inline .env:6]: Is there a specific reason for deciding to change the port?

File: src/backend/server.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
-const app = require('./API/teams');
+const app = require("./app");
 
-const PORT = process.env.PORT || 3030;
-app.listen(PORT, () => {
-  console.log(`Server running on port ${PORT}`);
-});
+const port = parseInt(process.env.PORT, 10) || process.env.API_PORT;
```
Comment (aina21) [inline src/backend/server.js:3]: Do you need to specify the API_PORT in the environment (.env) file?

File: src/component/CreateMember.jsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
```
Comment (aina21) [inline src/component/CreateMember.jsx]: This isn't directly related to the current PR. Perhaps we should consider adding it during the createMember implementation.

File: src/component/CreateTeam.jsx [hunk 4]
```diff
@@ -1,38 +1,40 @@
...
+// use this format if fetching
+//import { apiPath } from '../api';
+// fetch(apiPath('/teams'))
+const CreateTeam = ({ setTeamsDatabase }) => {
   const handleCreateTeam = async () => {
     try {
       const randomCode = Math.random().toString(36).substring(2, 8).toUpperCase();
 
-      const title = prompt("Enter new team name:");
+      const newName = prompt("Enter new team name:");
```
Comment (aina21) [inline src/component/CreateTeam.jsx:10]: is there reason for renaming title to newName?

File: src/component/MemberDetails.jsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react";
```
Comment (aina21) [inline src/component/MemberDetails.jsx]: This isn't directly related to the current PR. Make the new PR for this change.

File: .env (hunk 1 above)
Comment (shazkhan2) [reply .env:6]: We had issues with ports as it always complained about port already in use. Now backend is on 4051 and front on 3000 and its finally resolved.

File: src/component/CreateMember.jsx (hunk 3 above)
Comment (shazkhan2) [reply src/component/CreateMember.jsx]: I have deleted it and will make a new PR

File: src/component/CreateTeam.jsx (hunk 4 above)
Comment (shazkhan2) [reply src/component/CreateTeam.jsx:10]: We had used title in the database as column name for what was actaully a team name. To more accurately describe what it is, we change title to newName.

File: src/component/MemberDetails.jsx (hunk 5 above)
Comment (shazkhan2) [reply src/component/MemberDetails.jsx]: I have deleted it and will create a nwe PR for it.

**Pull request closed without merging**